### ISSUE TITLE
Test fails if sokar is instantiated multiple times 

### ISSUE THREAD

**Issue by ThomasObenaus** [ThomasObenaus/sokar#62]
Reason seams to be the global (singleton) metrics collector.

**Comment by ThomasObenaus** [comment]
# Stack Trace
```
--- FAIL: Test_SetupScaler (0.00s)
panic: duplicate metrics collector registration attempted [recovered]
	panic: duplicate metrics collector registration attempted [recovered]
	panic: duplicate metrics collector registration attempted

goroutine 10 [running]:
testing.tRunner.func1(0xc000136700)
	/usr/local/go/src/testing/testing.go:792 +0x387
panic(0xc70c60, 0xc000224ba0)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
github.com/thomasobenaus/sokar/vendor/github.com/golang/mock/gomock.(*Controller).Finish(0xc00025c180)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/vendor/github.com/golang/mock/gomock/controller.go:246 +0x24e
panic(0xc70c60, 0xc000224ba0)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc0000aa370, 0xc000248900, 0x1, 0x1)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus/registry.go:391 +0xad
github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus.MustRegister(0xc000248900, 0x1, 0x1)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus/registry.go:176 +0x53
github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus/promauto.NewCounterVec(0xd466cc, 0x5, 0xd4510d, 0x3, 0xd53a43, 0x17, 0xd70520, 0x84, 0x0, 0xc0002488c0, ...)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/vendor/github.com/prometheus/client_golang/prometheus/promauto/auto.go:146 +0xe3
github.com/thomasobenaus/sokar/metrics.NewWrappedCounterVec(0xd466cc, 0x5, 0xd4510d, 0x3, 0xd53a43, 0x17, 0xd70520, 0x84, 0x0, 0xc0002488c0, ...)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/metrics/wrappedProm.go:38 +0x8d
github.com/thomasobenaus/sokar/scaler.NewMetrics(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/scaler/metrics.go:22 +0xe2
github.com/thomasobenaus/sokar.setupScaler(0xd44ff3, 0x3, 0x0, 0x1, 0xd4f16c, 0x11, 0x1, 0xe155a0, 0xc000248530, 0xc000266000, ...)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/main.go:250 +0x362
github.com/thomasobenaus/sokar.Test_SetupScaler(0xc000136700)
	/home/winnietom/work/projects/golang/src/github.com/thomasobenaus/sokar/main_test.go:71 +0x579
testing.tRunner(0xc000136700, 0xd766c0)
	/usr/local/go/src/testing/testing.go:827 +0xbf
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:878 +0x35c
FAIL	github.com/thomasobenaus/sokar	0.014s
Error: Tests failed.
```

**Comment by ThomasObenaus** [comment]
See: https://github.com/ThomasObenaus/sokar/blob/master/main_test.go#L69

**Comment by ThomasObenaus** [comment]
Solved with #70
